data/zilliqa/assets/zil1kwfu3x9n6fsuxc4ynp72uk5rxge25enw7zsf9z/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "Score",
+	"symbol": "SCO",
+	"web": "https://www.uffsports.com",
+	"supply": "100000000",
+	"email": "info@uffsports.com",
+	"whitepaper": "https://docs.google.com/document/d/1ObJ5CxOs8k9TJssLUPdWLLM0EmZngpXY/edit",
+	"publicTeam": true,
+	"links": {
+		"telegram": "https://t.me/joinchat/SDRESrh77-LavqpT",
+		"twitter": "https://twitter.com/UFFSports",
+		"linkedin": "https://www.linkedin.com/company/ultimate-franchise-fantasy-sports"
+	},
+	"gen": {
+		"logo": "logo.svg"
```

Comment:
Honestly this is one of the rare cases where the logo should be in PNG given its number of elements which impact the size (37kb atm). Have you tried svg minify? If it doesn't impact size too much could you just convert it to a PNG?

Reply:
28.6 KB. Will that do?

Reply:
prefer to keep this in svg format, if possible

Reply:
Hum yeah should be ok

Reply:
Ok thanks

Reply:
Do you want to reject the PR and I create a new PR?

Reply:
No you can just update this one

Reply:
Ok